halo2_proofs/src/multicore.rs

```diff
@@ -1,9 +1,15 @@
 //! An interface for dealing with the kinds of parallel computations involved in
 //! `halo2`. It's currently just a (very!) thin wrapper around [`rayon`] but may
 //! be extended in the future to allow for various parallelism strategies.
-
+#[cfg(feature = "multicore")]
```

Comment:
Using `maybe-rayon`, I think this feature flag could just be removed and these imports left as unconditional (though we would need to keep `current_num_threads` separately conditional as I don't think `maybe-rayon` provides an equivalent).